tests/testthat/helper.R

```diff
@@ -1,31 +1,38 @@
-# This function behaves like Sys.getenv() but returns values from the
-# store of fake REDCap API credentials in inst/misc
+# This function returns a list of named credentials either with `Sys.getenv()` if `fake = FALSE`
+# or from `inst/misc/fake_credentials.csv` if `fake = TRUE`
+#
+# By default all credentials stored in `fake_credentials.csv` are returned
 
-get_fake_credentials <- function(credentials) {
+get_credentials <- function(credentials = NULL, fake = FALSE) {
```

Comment:
Is it at all worth moving this out to the `R/dev_utils.R` location so you don't have to copy it in two locations? Or is it necessary to have it in both locations for it to work properly?

Reply:
We could do that but the only catch is it would need to be an exported function to work in the vignette. I think other solutions like referring to it with `:::` upset R CMD Check